commit fbbe2012098459bfdb694b529f45ba119ec40085
Author: Gwenole Beauchesne <gbeauchesne@splitted-desktop.com>
Date:   Mon Apr 19 14:01:55 2010 +0200

    [G45] Don't export internal symbols as they can be mixed up
    with those from the system i965_dri.so.

diff --git a/i965_drv_video/Makefile.am b/i965_drv_video/Makefile.am
index 058b525..426ae18 100644
--- a/i965_drv_video/Makefile.am
+++ b/i965_drv_video/Makefile.am
@@ -22,7 +22,7 @@
 
 SUBDIRS = shaders
 
-AM_CFLAGS = -Wall -I$(top_srcdir) -I$(top_srcdir)/va -I$(top_srcdir)/va/x11 @DRM_CFLAGS@
+AM_CFLAGS = -Wall -I$(top_srcdir) -I$(top_srcdir)/va -I$(top_srcdir)/va/x11 @DRM_CFLAGS@ -fvisibility=hidden
 
 i965_drv_video_la_LTLIBRARIES = i965_drv_video.la
 i965_drv_video_ladir = @LIBVA_DRIVERS_PATH@
diff --git a/i965_drv_video/i965_drv_video.c b/i965_drv_video/i965_drv_video.c
index 936870f..1e88927 100644
--- a/i965_drv_video/i965_drv_video.c
+++ b/i965_drv_video/i965_drv_video.c
@@ -1848,6 +1848,10 @@ i965_Terminate(VADriverContextP ctx)
 
 VAStatus 
 __vaDriverInit_0_31(  VADriverContextP ctx )
+    __attribute__((__visibility__("default")));
+
+VAStatus 
+__vaDriverInit_0_31(  VADriverContextP ctx )
 {
     struct i965_driver_data *i965;
     int result;
